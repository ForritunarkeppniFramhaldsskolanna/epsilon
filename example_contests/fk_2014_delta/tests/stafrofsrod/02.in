16
uipksxrufyyjiwm
ynlyxuqkczhbeujlxn
wfooihoykbqdusyv
mstkayz
qs
ogfjtwbsyccwmjggnsdu
ruaipw
jal
uxgcbkvjxcyujzyjq
sf
womzqd
kgrb
maakiyh
rtmoaejzcqvvufx
bbkgpu
c
