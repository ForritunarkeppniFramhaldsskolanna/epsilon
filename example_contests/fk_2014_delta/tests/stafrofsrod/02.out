bbkgpu
c
jal
kgrb
maakiyh
mstkayz
ogfjtwbsyccwmjggnsdu
qs
rtmoaejzcqvvufx
ruaipw
sf
uipksxrufyyjiwm
uxgcbkvjxcyujzyjq
wfooihoykbqdusyv
womzqd
ynlyxuqkczhbeujlxn
